Dockerfile

```diff
@@ -7,18 +7,19 @@ RUN python -m venv --copies $VIRTUAL_ENV
 ENV PATH="$VIRTUAL_ENV/bin:$PATH"
 
 RUN python -m pip install --upgrade pip
+RUN pip install poetry==1.7.0
```

Comment:
Is it necessary to lock the version of this?

Reply:
No. It is recommended practice though.